src/components/NcSelect/NcSelect.vue

```diff
@@ -1263,12 +1269,13 @@ body {
 
 	.vs__selected-options {
 		// If search is hidden, ensure that the height of the search is the same
-		min-height: 40px; // 36px search height + 4px search margin
+		min-height: var(--default-clickable-area);
```

Comment:
Selector specificity either isn't big enough or scoped enough here (see leaked from server and viewer):
![image](https://github.com/user-attachments/assets/79998916-3d63-458d-82cb-43090f37d8ed)

Even with that applied, NcSelect height is 36px, not 34px

Reply:
There is #5883